### ISSUE TITLE
firebase_core_web not found. Please update settings.gradle.

### ISSUE THREAD

**Issue by AnandMG02** [FirebaseExtended/flutterfire#2796]
**Describe the bug**
while running flutter , i am getting the below error 

```
> Plugin project :firebase_core_web not found. Please update settings.gradle.
> Plugin project :cloud_firestore_web not found. Please update settings.gradle.
```

after i went through [https://github.com/FirebaseExtended/flutterfire/issues/2599](https://github.com/FirebaseExtended/flutterfire/issues/2599) , i updated the settings.gradle. Still am getting the same error .

**Comment by TahaTesser** [comment]
Hi @AnandMG02 
Are you running build? or run? 
Can you please provide your `flutter doctor -v` your `flutter run --verbose`  or `flutter build apk --verbose` 
Thank you

**Comment by luc-dotcom** [comment]
Try to target min android api to 21

**Comment by TahaTesser** [comment]
Hi @luc-dotcom 
Hi just tried to reproduce using target api 21 

<img width="723" alt="Screenshot 2020-06-19 at 8 46 32 PM" src="https://user-images.githubusercontent.com/48603081/85148701-12379280-b26e-11ea-81d5-01f742701784.png">

**Comment by zeh** [comment]
This happens on a fresh Flutter project, likely because of changes to `settings.gradle`.

1. Run `flutter create <name>`
2. Go through instructions for [adding Firebase to Android package](https://firebase.google.com/docs/flutter/setup?platform=android)
3. Try `flutter run`, get error
    ![image](https://user-images.githubusercontent.com/49529/85230563-0d362880-b3bf-11ea-82dd-e25df1d09e9f.png)

This error happens even with the addition of `firebase_core` only. Apparently [it's a 4.3+ issue](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_core/firebase_core_web/README.md).

The app does run, but this is probably indicative of an error somewhere.

**Comment by Salakar** [comment]
Hey, as part of our on-going work for #2582, this should have been resolved in our Firebase Core rework (#2890) - which has now been merged into master. We'll look at publishing some prereleases in the next few days. Thank you

**Comment by HeroCodes770** [comment]
@Salakar has the issue been resolved now. Mine application cannot connect to firestore and gives the same error firebase_core_web not found
